### PR TITLE
Remove upgrade tests env vars

### PR DESCRIPTION
**Pull request by mambocab**
@ptnapoleon and @knifewine to review.

Please review carefully to make sure the changes are correct. Correctness here means that the changed code executes as though `QUERY_UPGRADED=true`, `QUERY_OLD=true`, and `UPGRADE_MODE=normal`.

### CODE REVIEW

File: upgrade_tests/upgrade_base.py [hunk 1]
```diff
@@ -256,12 +219,4 @@ def get_node_version(self, is_upgraded):
         return max(node_versions) if is_upgraded else min(node_versions)
 
     def tearDown(self):
-        # Ignore errors before upgrade on Windows
-        # We ignore errors from 2.1, because windows 2.1
-        # support is only beta. There are frequent log errors,
-        # related to filesystem interactions that are a direct result
-        # of the lack of full functionality on 2.1 Windows, and we dont
-        # want these to pollute our results.
-        if is_win() and UPGRADE_MODE != "all" and self.cluster.version() <= '2.2':
```
Comment (mambocab) [inline upgrade_tests/upgrade_base.py]: Yes, thank you.

**Comment by mambocab** [comment]
I've pushed a fix to a comment, which I should squash into  the third commit in this PR before merge.

**Comment by ptnapoleon** [comment]
Lgtm now

**Comment by mambocab** [comment]
Thanks. I'm waiting for review from hatch as well, just to get a lot of eyes on it, since this feels very mess-up-able.

**Comment by knifewine** [comment]
+1 lgtm

**Comment by mambocab** [comment]
Alllllllllrighty, thanks for the PRs y'all. Merging.

**Pull request merged**